<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>961</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="10,30,40">
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="dir_edit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="dir_label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Current directory</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>dir_edit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="output_label">
       <property name="text">
        <string>Output</string>
       </property>
       <property name="buddy">
        <cstring>output_view</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QGraphicsView" name="output_view"/>
     </item>
     <item row="1" column="0" rowspan="2" colspan="2">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="analconv_tab">
        <attribute name="title">
         <string>Analyse Convergence</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>311</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="analconv_layout" stretch="0,0,0,0,0,0">
          <item>
           <widget class="QRadioButton" name="analconv_1">
            <property name="text">
             <string>Check orthonormality of SPFs in Psi file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analconv_2">
            <property name="text">
             <string>Check orthonormality of SPFs in Restart file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analconv_3">
            <property name="text">
             <string>Plot populations of natural orbitals</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analconv_4">
            <property name="text">
             <string>Plot populations of grid edges</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analconv_5">
            <property name="text">
             <string>Plot time-evolution of norm of wavefunction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analconv_6">
            <property name="text">
             <string>Norm of wavefunction on restart file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="analconv_push">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>451</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="analint_tab">
        <attribute name="title">
         <string>Analyse Integrator</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>311</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="analint_layout" stretch="0,0,0,0">
          <item>
           <widget class="QRadioButton" name="analint_1">
            <property name="text">
             <string>Analyse Step Size</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analint_2">
            <property name="text">
             <string>Look at timing file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analint_3">
            <property name="text">
             <string>Type update file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analint_4">
            <property name="text">
             <string>Plot update step size</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="analint_push">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>451</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="analres_tab">
        <attribute name="title">
         <string>Analyse Results</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>311</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="analres_layout" stretch="0,0,0,0">
          <item>
           <widget class="QRadioButton" name="analres_1">
            <property name="text">
             <string>Plot autocorrelation function</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analres_2">
            <property name="text">
             <string>Plot Fourier transform of autocorrelation function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analres_3">
            <property name="text">
             <string>Plot spectrum from autocorrelation function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analres_4">
            <property name="text">
             <string>Plot eigenvalues from matrix diagonalisation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="analres_push">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>451</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="analevol_tab">
        <attribute name="title">
         <string>Analyse System Evolution</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>311</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="analevol_layout" stretch="0,0,0">
          <item>
           <widget class="QRadioButton" name="analevol_1">
            <property name="text">
             <string>Plot 1D density evolution</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analevol_2">
            <property name="text">
             <string>Plot 2D density evolution</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="analevol_3">
            <property name="text">
             <string>Plot diabatic state population</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="analevol_push">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>451</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="analpes_tab">
        <attribute name="title">
         <string>Analyse Potential Surface</string>
        </attribute>
        <widget class="QPushButton" name="analpes_push">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>451</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="compcal_tab">
        <attribute name="title">
         <string>Compare Calculations</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>dir_edit</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>analconv_1</tabstop>
  <tabstop>analconv_2</tabstop>
  <tabstop>analconv_3</tabstop>
  <tabstop>analconv_4</tabstop>
  <tabstop>analconv_5</tabstop>
  <tabstop>analconv_6</tabstop>
  <tabstop>output_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
